src/Data/AudienceFolder.elm

```diff
@@ -7,10 +10,14 @@ This module implements everything related to audience folder resource.
 
 # Interface
 
-@docs AudienceFolder, audienceFoldersJSON
+@docs AudienceFolder, fixtures
```

Comment:
Looks like the doc comment is a bit useless given that we don't `elm make --docs` in our production codebase :thinking:

(I wanted to comment on the fact that `decoder` fn is missing from the doc comment, but I don't think it's a very big issue -- precisely because we don't generate the docs...)